position_sender_node:
  ros__parameters:
    real_position_topic: real_position
    short_motor_srv: short_motor
    update_position_topic: update_position
    update_period: 0.1

main_robot:
  zynq_simu_node:
    ros__parameters:
      rcv_comm_topic: raspi_zync
      short_motor_srv: short_motor
      send_comm_topic: zync_raspi
      robot_components: ["motors"]

  motor_simu_node:
    ros__parameters:
      real_position_topic: real_position
      short_motor_srv: short_motor
      random_moves: False
      boundaries:  [0.0, 3.0, 0.0, 2.0]
      compute_period: 0.05
      angular_velocity: 0.4
      linear_velocity: 0.15

  lidar_simu_node:
    ros__parameters:
      object_topic: raw_obstacles
      scan_topic: scan
      update_position_topic: update_position
      color_service: init/team_color
      default_color: yellow
      available_colors: ["yellow", "blue"]
      beacons: [-0.094, 0.072, -0.094, 1.928, 3.094, 1., 1.275, 2.12]
      boundaries:  [0.0, 3.0, 0.0, 2.0]

      use_lidar_points: true
      lidar_range: 10.
      radius: 0.05
      num_points: 600
      angle_info: [0., 360.]
